zohar.js

```diff
@@ -0,0 +1 @@
+console.log("Hi Zohar")
```

Comment:
very niiiiiiiiiiiiiiiiiiiiice
```suggestion
console.log("Hi Zohar Pfeffer")
```